packages/config-utils/chunk-mapper.js

```diff
@@ -1,35 +1,59 @@
 class ChunkMapper {
   constructor(options) {
+    const { _unstableHotReload, ...rest } = options;
     this.config = {};
-    this.options = options || {};
+    this.options = rest || {};
+    this._unstableHotReload = _unstableHotReload;
+    this.hotChunkRegex = RegExp('\\.hot-update\\.js$');
```

Comment:
Can we add an option to customize this? By default we'd use the hot-update regexp, but we could add option to customize it.

Reply:
not sure if there is anything to customize :thinking: . Webpack will always call the hot-update chunks the same.

Reply:
If that's the thing I don't think we have to initialize the value on constructor level. But rather have it outside constructor.

Reply:
Done

Reply:
I meant more like a class member, but this works as well :smiley:

Reply:
Yeah sorry :D it does the same thing anyways